### PR TITLE
chore: bump op-rs

### PR DESCRIPTION
**Pull request by razvan**
# Description

part of: https://github.com/stackabletech/issues/issues/640

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] CRD documentation for all fields, following the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs/style-guide).
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
- [ ] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] Code contains useful logging statements
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs/style-guide).
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

**Comment by razvan** [comment]
CI https://testing.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/superset-operator-it-custom/2/

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,8 +12,13 @@
   - `podOverrides`
   - `affinity`
 
+### Fixed
```
Comment (sbernauer) [inline CHANGELOG.md:15]: I'd be in favor of removing the changelog, as this operator is not using S3 at all

**Review by sbernauer — CHANGES_REQUESTED** [review]
Only a changelog remark

File: CHANGELOG.md (hunk 1 above)
Comment (razvan) [reply CHANGELOG.md]: done

**Review by sbernauer — APPROVED** [review]

**Pull request merged**